osutil/cmdline.go

```diff
@@ -17,13 +17,29 @@
  *
  */
```

Comment:
in the context of osutil, is not clear whether command line is the process or the kernel one. so I think this file should be called "kcmdline.go" for a bit more clarity